### ISSUE TITLE
WCAG 2.2 AA updates to guidance to comply, and help teams to comply, with new criteria

### ISSUE THREAD

**Issue by calvin-lau-sig7** [alphagov/govuk-design-system#2729]
## What

Update component and pattern guidance to include any new requirements service teams need to follow to comply with WCAG 2.2 AA.

This epic specifically covers component and pattern guidance that needs to be added, such as specific instructions and requirements for particular components.

This also covers 'styles' however audit seems to show it as mostly unaffected, *except* for focus states which is currently being drafted.

This comes under: [#3040](https://github.com/alphagov/govuk-frontend/issues/3040) and will be done alongside: #2448 

[About 54 updates in total](https://docs.google.com/document/d/1nIxIKgVUheC8zvVdqWIcqtVPCl7oTN4hv11R9rDwdvM/edit?usp=sharing):

## Epic lead
Calvin

## Why

WCAG 2.2 is set to be published in 2023. As such, we hope to bring the GOV.UK Design System website in line with WCAG 2.2 criteria within 9 months of the publish date.

## Who needs to work on this

Accessibility specialist, Content designers, interaction designer

## Who needs to review this

Developers, interaction designers

```[tasklist]
- [x] Audited components and patterns to check for WCAG 2.2 AA compliance — [Google Sheet: WCAG 2.2 assessment of DS elements](https://docs.google.com/spreadsheets/d/194aFvTTi4JlyEw4b21dHmCn5Iry4t4RY8A8VOKydFj0/edit?usp=sharing)
- [x] Brief written for new content to be added (job stories) — [Google Doc: WCAG 2.2 content updates](https://docs.google.com/document/d/1nIxIKgVUheC8zvVdqWIcqtVPCl7oTN4hv11R9rDwdvM/edit?usp=sharing)
- [x] "First 15" Content drafted, including proposed copy and location
- [x] "First 15" Draft reviewed by accessibility expert
- [ ] #3063
- [x] "Next 30" Content drafted, including proposed copy and location
- [x] "Next 30" Draft reviewed by accessibility expert
- [ ] https://github.com/alphagov/govuk-design-system/issues/3064
- [ ] https://github.com/alphagov/govuk-design-system/issues/3276
- [ ] https://github.com/alphagov/govuk-design-system/issues/3304
- [ ] https://github.com/alphagov/govuk-design-system/issues/3302
- [ ] https://github.com/alphagov/govuk-design-system/issues/3338
- [ ] https://github.com/alphagov/govuk-design-system/issues/3311
- [ ] Rebase with GOV.UK Frontend v5.0 (see https://gds.slack.com/archives/C011V7RAAKY/p1701695706964249?thread_ts=1701694882.286329&cid=C011V7RAAKY)
- [ ] Published to production (live website)
```

```[tasklist]
### 'DUPLICATED' issues that were already in our doc
- [ ] https://github.com/alphagov/govuk-design-system/issues/3008
- [ ] https://github.com/alphagov/govuk-design-system/issues/2882
- [ ] https://github.com/alphagov/govuk-design-system/issues/2866
- [ ] https://github.com/alphagov/govuk-design-system/issues/2878
- [ ] https://github.com/alphagov/govuk-design-system/issues/2871
```

**Comment by calvin-lau-sig7** [comment]
Re: how we highlight this new content to users, a few of us working on this agreed we'll want to create some sort of 'summary page' to index and show this content. Use case is for teams that want to know what changes they need to make to existing work to comply with WCAG 2.2.
